### PR TITLE
add  r-coin  r-formattable  r-gistr  r-hmisc  r-igraph  r-lsmeans  r-tidytext

### PR DESCRIPTION
**Pull request by bgruening**
r-coin
 r-formattable
 r-gistr
 r-hmisc
 r-igraph
 r-lsmeans
 r-tidytext

xref: https://github.com/conda-forge/staged-recipes/issues/2009
ping @johanneskoester 

Level 6!

### CODE REVIEW

**Comment by jakirkham** [comment]
Toggling to restart CIs.

File: recipes/r-igraph/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
+    - r-matrix
+    - r-nmf
+    - r-irlba
+    - r-magrittr
+    - libgcc  # [not win]
```
Comment (isuruf) [inline recipes/r-igraph/meta.yaml]: You need `gmp` for build and run. (For build, `gmp` is pulled in by the `gcc` package)

File: recipes/r-igraph/meta.yaml [hunk 2]
```diff
@@ -40,6 +42,7 @@ requirements:
     - r-irlba
     - r-magrittr
     - libgcc  # [not win]
+    - gmp >=5.0.1,<7
```
Comment (isuruf) [inline recipes/r-igraph/meta.yaml]: Use `{{native}}gmp` instead of just `gmp`

File: recipes/r-igraph/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+source:
+  fn: igraph_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/igraph_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/igraph/igraph_{{ version }}.tar.gz
+  sha256: dc64ed09b8b5f8d66ed4936cde3491974d6bc5178dd259b6eab7ef3936aa5602
+
+build:
+  number: 0
+  # compilation errors on windows
```
Comment (isuruf) [inline recipes/r-igraph/meta.yaml]: What are the errors you get?

Comment (bgruening) [inline recipes/r-igraph/meta.yaml]: I don't have a copy but it was unrelated to gmp afaik.

**Comment by bgruening** [comment]
@isuruf green!!! :)

**Comment by jakirkham** [comment]
Restarted AppVeyor to see if the transient issue (problems downloading some source code) with one of the builds would go away.

**Comment by bgruening** [comment]
@jakirkham AppVeyor is building 3 times the same thing as we do not have Python packages. py2, py45 and py36. So two tests were passing. I think it save to merge.

**Comment by bgruening** [comment]
@jakirkham green :)

**Pull request merged**